scipy/optimize/_lsq/lsq_linear.py

```diff
@@ -202,7 +210,7 @@ def lsq_linear(A, b, bounds=(-np.inf, np.inf), method='trf', tol=1e-10,
     >>> res = lsq_linear(A, b, bounds=(lb, ub), lsmr_tol='auto', verbose=1)
     # may vary
     The relative change of the cost function is less than `tol`.
-    Number of iterations: 16, initial cost: 1.5039e+04,final cost 1.1112e+04, first-order optimality 4.66e-08.
+    Number of iterations 16, initial cost 1.5039e+04, final cost 1.1112e+04, first-order optimality 4.66e-08.
```

Comment:
Preferably. pep8 should complain about this, too.

Aside from style, IPython's docstring display tends to get confused when scrolling across a long line, and it might omit a line until you hit <kbd>Ctrl</kbd>+<kbd>L</kbd> a few times.